custom/samveg/case_importer/validators.py

```diff
@@ -60,7 +60,7 @@ def _validate_mandatory_columns(cls, spreadsheet):
 
 class MandatoryValueValidator(BaseValidator):
     @classmethod
-    def run(cls, row_num, raw_row, fields_to_update):
+    def run(cls, row_num, raw_row, fields_to_update, import_context):
```

Comment:
Why not make `import_context` an optional kwarg?

Reply:
https://github.com/dimagi/commcare-hq/pull/30928/commits/ea5f4f63d7ebff99cac25439dbef2ffae3b2988a